client/signup/steps/intent/intent-screen.scss

```diff
@@ -0,0 +1,66 @@
+@import '@wordpress/base-styles/_breakpoints.scss';
+@import '@wordpress/base-styles/_mixins.scss';
+
+.intent-screen__cards {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	margin-top: -24px;
+	margin-bottom: 40px;
+	padding-left: 40px;
+	
+	@include break-small {
+		padding-left: 60px;
+	}
+}
+
+.intent-screen__card {
+	display: flex;
+	align-items: flex-start;
+	position: relative;
+	padding: 24px 0;
+	
+	@include break-small {
+		width: 342px;
+	}
+
+	@include break-medium {
+		width: 456px;
+	}
+
+	&-icon {
+		position: absolute;
+		top: 24px;
+		left: -40px;
+		@include break-small {
+			left: -48px;
+		}
+	}
+
+	&-info {
+		flex: 1;
+		padding-right: 6px;
+	}
+
+	&-title {
+		margin-bottom: 8px;
+		font-size: $font-title-small;
+		line-height: 1.2;
+		color: #101517;
+	}
+
+	&-description {
+		font-size: $font-body-small;
+		color: #646970;
+		margin-bottom: 0;
+	}
+
+	&-button {
+		border-radius: 4px; /* stylelint-disable-line */
+		min-width: 130px;
```

Comment:
If we change to another language, using `min-width` to make buttons have the same width might not work. But I don't have a better idea now 😅 The only way I think is to put all text as invisible inside the button so it can have the same width and we use absolute positioning to show the current one.